config/routes.rb

```diff
@@ -22,10 +22,11 @@
     resources :orders
   end
 
-  scope ":slug", module: "tenants", as: "tenant" do
-    get "/" => "items#index"
+  scope ':slug', module: 'tenants', as: 'tenant' do
+    get '/' => 'items#index'
     resources :items
+    get '/dashboard' => 'dashboard#show'
   end
 
-  match '/create_order', via: [:get], to: "orders#create"
+  match '/create_order', via: [:get], to: 'orders#create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.